salt/states/zenoss.py

```diff
@@ -0,0 +1,71 @@
+# -*- coding: utf-8 -*-
+'''
+State to manage monitoring in Zenoss.
+
```

Comment:
Gotcha. Updated and pushed as dcd7775d04f491919025841906909045dd65ff72

Thanks for the quick feedback!